### ISSUE TITLE
Line breaks in photo/document caption are broken

### ISSUE THREAD

**Issue by kitsunyan** [mukel/telegrambot4s#49]
I've updated library from `3.0.4` to `3.0.13` and faced the problem with line breaks in document title. They are sent as plain text.

```scala
SendDocument(chatId, InputFile("file.png", byteArray),
  caption = Some("first line\nsecond line"))
```

Resulting message in `3.0.4`:
```
first line
second line
```

In `3.0.13`:
```
first line\nsecond line
```

**Comment by mukel** [comment]
Thanks for reporting, this is yet another serialization issue, working on a fix, stay tuned.

**Comment by mukel** [comment]
Fixed in v3.0.14 please update. These serialization bugs are hard to track, but easy to fix, thanks for reporting, do not hesitate to report further issues/bugs.

**Comment by kitsunyan** [comment]
Great, it works. Thanks for your work on this library!
